src/controls/extras/tree.coffee

```diff
@@ -170,6 +170,9 @@ class Tree extends Typed
           setTimeout(arguments.callee, 0)
 
     timer = false
+    bandsCountId = "bands-count-" + $('.bands-count').length
```

Comment:
I agree with Bo.

Is there any reason why `bandsCountId`, `bandsFilterId` and `selectedBandsId` would be different? Just use the Id that gets created with the parent (`SMAPL1L2-bandSubsetOptions-47`)